scapy/contrib/gtp_v2.py

```diff
@@ -616,7 +658,7 @@ class IE_Cause(gtp.IE_Base):
 class IE_RecoveryRestart(gtp.IE_Base):
     name = "IE Recovery Restart"
     fields_desc = [ByteEnumField("ietype", 3, IEType),
-                   ShortField("length", 5),
+                   ShortField("length", None),
```

Comment:
Why are you changing this value?

Reply:
Same comment as - https://github.com/secdev/scapy/pull/2488#discussion_r386762044